code/obj/item/storage/backpack_belt_etc.dm

```diff
@@ -443,6 +443,19 @@
 		STOP_TRACKING_CAT(TR_CAT_NUKE_OP_STYLE)
 		..()
 
+/obj/item/storage/fanny/janny
```

Comment:
Would change this to be a child of the belt item and give it a can_hold list of all the misc janitorial stuff. Having janitors spawn with a 7-slot belt item seems pretty powerful.

Reply:
I see your concern, however I don't like the idea of making it a belt but having it look and read like a fanny pack. It will make it confusing for people what's actually allowed to fit in there. I'd rather just make it a 5 slot item, it'll still be a massive improvement for janitors.